regression/scripts/noise_optimization/Procedure.py

```diff
@@ -440,7 +440,7 @@ def noise_approach(nexus):
     mission = nexus.missions.landing
     nexus.analyses.landing.noise.settings.approach = 1    
     results = nexus.results
-    #results.approach = mission.evaluate()
+    results.approach = mission.evaluate()
```

Comment:
Just wondering if we needed results.approach from before? We need to look at this.

Reply:
We don't  need results.approach every time, just to update the regression.  commented out.